docs/en/ingest-management/release-notes/release-notes-8.12.asciidoc

```diff
@@ -132,6 +132,50 @@ So, if you are relying on an integration level custom ingest pipeline introduced
 Refer to the <<data-streams-pipelines,Ingest pipelines>> documentation for details and examples.
 ====
 
+[discrete]
+[[known-issues-8.12.1]]
+=== Known issues
+
+[[known-issue-3263-8121]]
+.Agents upgraded to 8.12.0 are stuck in a non-upgradeable state
+[%collapsible]
+====
+
+*Details*
+
+An issue discovered in {fleet-server} prevents {agents} that have been upgraded to version 8.12.0 from being upgraded again, using the {fleet} UI, to version 8.12.1 or higher.
+
+*Impact* +
+
+As a workaround, we recommend you to use the {kib} {fleet} API to upgrade these agents using the `force` flag.
```

Comment:
hey @kilfoyle , I noticed that the workaround in 8.12.1 was not updated (incorrectly states the upgrade with force flag): https://www.elastic.co/guide/en/fleet/current/release-notes-8.12.1.html
Could we correct this?

Another request, could we add at the end of the workaround something like this:
`Wait at least 10 minutes, and then the agents should be upgradeable again.`

Reply:
Ah! My mistake. Thanks for catching that. I've opened a new PR to update the 8.12.1 release notes, and also to add your suggested text to the workaround steps (in both 8.12.0 and 8.12.1).

https://github.com/elastic/ingest-docs/pull/918